### ISSUE TITLE
Carthage update fails to build PVNES-watchOS

### ISSUE THREAD

**Issue by Fahrenheit** [giginet/NESEmulator-watchOS#2]
Still getting the same issue. Here's the output.

*** Fetching FCEUX-watchOS
*** Checking out FCEUX-watchOS at "0.1.0"
*** xcodebuild output can be found in /var/folders/12/yk645t6d2f598_qyw6468v200000gn/T/carthage-xcodebuild.umVJ7l.log
*** Building scheme "PVSupport-watchOS" in FCEUX.xcworkspace
*** Building scheme "PVNES-watchOS" in FCEUX.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/blank/Carthage/Checkouts/FCEUX-watchOS/FCEUX.xcworkspace -scheme PVNES-watchOS -configuration Release -derivedDataPath /Users/blank/Library/Caches/org.carthage.CarthageKit/DerivedData/9.2_9C40b/FCEUX-watchOS/0.1.0 -sdk watchos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/12/yk645t6d2f598_qyw6468v200000gn/T/FCEUX-watchOS SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO (launched in /Users/blank/Carthage/Checkouts/FCEUX-watchOS)

**Comment by giginet** [comment]
Sorry for my late reply.
I just released FCEUX-watchOS 0.2.0.

Write `Cartfile` like following to try newer version.

```
github "giginet/FCEUX-watchOS" "0.2.0"
```
  
```console
$ carthage update --platform watchOS
```

**Comment by giginet** [comment]
I updated dependencies on #3.
Could you try again?
